common/global.h

```diff
@@ -10,6 +10,7 @@ struct mpv_global {
     struct mp_client_api *client_api;
     char *configdir;
     struct stats_base *stats;
+    struct demux_packet_pool *packet_pool;
```

Comment:
Putting this in global feels a bit wrong. Can't this just live in `demux.h` somewhere?

Reply:
This is pool of packets that live with MPContext. It has to outlive any demuxer insteance for it to be useful.